update readme to be more specific on where gradle.properties is (#9)
ministryofjustice/laa-data-access-api: README.md

Specify the Note field, e.g. “Token to allow access to LAA Gradle plugin”

> If you haven’t got a gradle.properties file create one under `~/.gradle/gradle.properties` in home directory(not project directory).

Reviewer: ```suggestion
If you don't already have one, create a `gradle.properties` file in your home directory at `~/.gradle/gradle.properties`.
```

I think I would reword it like this, as `~` already implies home directory
